libreflexxestype2/CMakeLists.txt

```diff
@@ -21,26 +21,22 @@ if(SUPPORTED MATCHES "true")
  
   # Build Reflexxes library with a Make file
   add_custom_command(
-    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/static/libReflexxesTypeII.a
+    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so
     COMMAND make -C "${CMAKE_CURRENT_SOURCE_DIR}/Linux" clean64 all64 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}  
   )
 
   # Export Reflexxes library for ROS
-  add_library(reflexxestype2 ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/static/libReflexxesTypeII.a)
+  add_library(reflexxestype2 ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so)
   target_include_directories(reflexxestype2 PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
     "$<INSTALL_INTERFACE:include>"
   )
-  ament_target_dependencies(reflexxestype2)
   set_target_properties(reflexxestype2 PROPERTIES LINKER_LANGUAGE CXX)
 
-  ament_export_include_directories(include)
-  ament_export_targets(export_reflexxestype2 HAS_LIBRARY_TARGET)
-  ament_export_dependencies()
-
   # Install header files
   install(DIRECTORY include/ DESTINATION include/${PROJECT_NAME}/)
   # TODO(andyz): This extra installation step shouldn't be necessary. Look into deleting it.
+  # (destogl): This is not possible, except for adding "libreflexxestype2/" to all reflexxes includes
```

Comment:
nitpick: go ahead and delete my TODO and just leave your comment. Thanks for verifying.

`# Install an extra set of header files to avoid adding "libreflexxestype2/" to all reflexxes includes`